uploader/s3/s3.go

```diff
@@ -30,7 +30,7 @@ func (u *Uploader) Upload(source string, destination string, contType string) er
 	}
 	defer reader.Close()
 
-	timeout, _ := time.ParseDuration("4m")
+	timeout := 4 * time.Minute
```

Comment:
should this be configurable? This might have been the reason why it was parsing? Not required for now, but let's think if this is something we need to support.

Reply:
this should probably be configurable but wasnt.  So i just cleaned up how it looked